src/widget/wtrackmenu.cpp

```diff
@@ -914,6 +915,18 @@ class ImportMetadataFromFileTagsTrackPointerOperation : public mixxx::TrackPoint
 
 } // anonymous namespace
 
+void WTrackMenu::slotUpdateReplaygainFromDeckGain() {
+    DEBUG_ASSERT(m_pTrack);
```

Comment:
I suggest to use VERIFY_OR_DEBUG_ASSERT for both precondition checks here.

Reply:
done